--- boot.ini	2020-03-13 21:24:54.000000000 +0000
+++ boot.ini.new	2020-03-15 12:11:59.000000000 +0000
@@ -8,7 +8,7 @@
 # setenv m "480x272p60hz"	# 480x272
 # setenv m "480x800p60hz"	# 480x800
 # setenv m "vga"          	# 640x480
-# setenv m "480p"         	# 720x480
+setenv m "480p"         	# 720x480
 # setenv m "576p"         	# 720x576
 # setenv m "800x480p60hz" 	# 800x480
 # setenv m "800x600p60hz" 	# 800x600
@@ -24,7 +24,7 @@
 # setenv m "1080i50hz"          # 1080I@50Hz
 # setenv m "1080p24hz"          # 1080P@24Hz
 # setenv m "1080p50hz"          # 1080P@50Hz
-setenv m "1080p"                # 1080P@60Hz
+# setenv m "1080p"                # 1080P@60Hz
 # setenv m "1920x1200"    	# 1920x1200
 
 # HDMI DVI Mode Configuration
@@ -59,13 +59,12 @@
 # UHS Card Configuration
 # Uncomment the line below to __DISABLE__ UHS-1 MicroSD support
 # This might break boot for some brand models of cards.
-setenv disableuhs "disableuhs"
-
+# setenv disableuhs "disableuhs"
 
 # Disable VPU (Video decoding engine, Saves RAM!!!)
 # 0 = disabled
 # 1 = enabled
-setenv vpu "1"
+setenv vpu "0"
 
 # Disable HDMI Output (Again, saves ram!)
 # 0 = disabled
@@ -73,19 +72,19 @@
 setenv hdmioutput "1"
 
 # Default Console Device Setting
-# setenv condev "console=ttyS0,115200n8"        # on serial port
+setenv condev "console=ttyS0,115200n8"        # on serial port
 # setenv condev "console=tty0"                    # on display (HDMI)
-setenv condev "console=tty0 console=ttyS0,115200n8"   # on both
+# setenv condev "console=tty0 console=ttyS0,115200n8"   # on both
 
 # Enable/Disable ODROID-VU7 Touchscreen
-setenv disable_vu7 "false" # false
+setenv disable_vu7 "true" # false
 
 # CPU Max Frequency
 # Possible Values: 96 192 312 408 504 600 720 816
 # 1008 1200 1320 1488 1536 1632 1728 and 1824
-setenv max_freq "1536"
+# setenv max_freq "1536"
 # setenv max_freq "1632"
-# setenv max_freq "1728"
+setenv max_freq "1728"
 # setenv max_freq "1824"
 
 ###########################################
@@ -95,7 +94,7 @@
 if test "${disable_vu7}" = "false"; then setenv hid_quirks "usbhid.quirks=0x0eef:0x0005:0x0004"; fi
 
 # Boot Arguments
-setenv bootargs "root=UUID=e139ce78-9841-40fe-8823-96a304a09859 rootwait rw ${condev} no_console_suspend fsck.repair=yes vdaccfg=0xa000 logo=osd1,loaded,0x7900000,720p,full dmfc=3 cvbsmode=576cvbs hdmimode=${m} m_bpp=${m_bpp} vout=${vout_mode} ${disableuhs} ${hdmi_hpd} ${hdmi_cec} ${enabledac} monitor_onoff=${monitor_onoff} max_freq=${max_freq} ${hid_quirks} fsck.repair=yes net.ifnames=0"
+setenv bootargs "root=UUID=e139ce78-9841-40fe-8823-96a304a09859 rootwait rw ${condev} no_console_suspend fsck.repair=yes vdaccfg=0xa000 logo=osd1,loaded,0x7900000,480p,full dmfc=3 cvbsmode=480cvbs hdmimode=${m} m_bpp=${m_bpp} vout=${vout_mode} ${disableuhs} ${hdmi_hpd} ${hdmi_cec} ${enabledac} monitor_onoff=${monitor_onoff} max_freq=${max_freq} ${hid_quirks} fsck.repair=yes net.ifnames=0 consoleblank=0 quiet loglevel=3 rd.systemd.show_status=false udev.log-priority=3"
 
 # Booting
 fatload mmc 0:1 0x21000000 uImage
