--- lr-flycast.sh	2020-02-22 20:26:49.184510817 +0000
+++ lr-flycast_odroid.sh	2020-02-22 20:33:45.350850265 +0000
@@ -14,7 +14,7 @@
 rp_module_help="Previously named lr-reicast then lr-beetle-dc\n\nDreamcast ROM Extensions: .cdi .gdi .chd, Naomi/Atomiswave ROM Extension: .zip\n\nCopy your Dreamcast/Naomi roms to $romdir/dreamcast\n\nCopy the required Dreamcast BIOS files dc_boot.bin and dc_flash.bin to $biosdir/dc\n\nCopy the required Naomi/Atomiswave BIOS files naomi.zip and awbios.zip to $biosdir/dc"
 rp_module_licence="GPL2 https://raw.githubusercontent.com/libretro/flycast/master/LICENSE"
 rp_module_section="opt"
-rp_module_flags="!mali !armv6"
+rp_module_flags="!armv6"
 
 function depends_lr-flycast() {
     local depends=()
@@ -46,6 +46,11 @@
             params+=("platform=rpi")
         fi
     fi
+    
+    if isPlatform "odroid-c1"; then
+        params+=(platform=odroid)
+    fi
+    
     # temporarily disable distcc due to segfaults with cross compiler and lto
     DISTCC_HOSTS="" make "${params[@]}"
     md_ret_require="$md_build/flycast_libretro.so"
